python/tests/reward_functions.py

```diff
@@ -0,0 +1,22 @@
+def fixed_reward(**kwargs):
+    return 1
+
+
+def constant_reward(**kwargs):
+    reward = kwargs["reward"]
+    chosen_action = kwargs["chosen_action"]
+    return reward[chosen_action - 1]
+
+
+def fixed_reward_two_action(**kwargs):
+    chosen_action = kwargs["chosen_action"]
+    chosen_context = kwargs["chosen_context"]
+    if chosen_context == 1 and chosen_action == 2:
```

Comment:
better to expose this logic to config file somehow:
1) either via more descriptive function name
2) or via extra parameter (like reward matrix)

Reply:
but let's do it in next Pr